src/test/java/com/easyrest/tests/LoginTest.java

```diff
@@ -14,19 +18,20 @@ public void testLogin() throws InterruptedException {
         //ExtentSparkReporter spark = new ExtentSparkReporter("src/test/Reports/loginTest_28.06.2022");
```

Comment:
You need this line

Reply:
> You need this line

this test is a draft, i think i should not even push that. We should delete this class in the nearest future